src/diffuse/dialogs.py

```diff
@@ -33,8 +33,9 @@
 
 # the about dialog
 class AboutDialog(Gtk.AboutDialog):
-    def __init__(self) -> None:
+    def __init__(self, parent):
```

Comment:
I'm sorry, but I can't approve that you remove type hints. Typeless source code is unmaintainable and I'm slowly (and painfully) adding type hints across the source code. How about that?

```suggestion
    def __init__(self, parent: Gtk.Widget) -> None:
```